test/e2e/testnet/testnet.go

```diff
@@ -367,7 +367,7 @@ func (t *Testnet) Start() error {
 				return fmt.Errorf("failed to start node %s", node.Name)
 			}
 			fmt.Printf("node %s is not synced yet, waiting...\n", node.Name)
-			time.Sleep(100 * time.Millisecond)
+			time.Sleep(1000 * time.Millisecond)
```

Comment:
[nit] seems more readable
```suggestion
			time.Sleep(10 * time.Second)
```

Reply:
I have updated the unit from milliseconds  to seconds.
Since this loop iterates 10 times and checks the node's status in each iteration, to achieve a total wait time of 10 seconds, the wait time for each iteration should be adjusted to 1 second.